src/pages/City/components/NearestCitiesWeatherWidget/NearestCitiesWeatherWidget.tsx

```diff
@@ -16,7 +17,13 @@ const NearestCitiesWeatherWidget = ({ cityName, coordinates }: Props) => {
     }
 
     if (data) {
-      return data.map((city) => <CityWeatherShortcut key={city.id} cityName={city.tags.name} />);
+      return data.map((city) => (
+        <CityWeatherShortcut
+          key={city.id}
+          cityName={city.tags.name}
+          button={<FavoritesButton cityName={city.tags.name} size={20} />}
```

Comment:
**question & suggestion** I'm not sure if its good solution to pass whole button as props. Maybe it can be solved by creating 2 components that differ from each other only by the button, like we did in `WeatherAqiWidgetLoader` and `WeatherInfoWidgetLoader` with `data-testid`

Reply:
I decided to create reusable component, because there is no point in writing the same code and change only button in my opinion. Maybe mentors can say more about which approach is correct

Reply:
yeah right, i think we should stick with your solution

Reply:
i will wait with resolve until mentors approve, so they can tell their opinions on this topic

Reply:
Where that reusable component will be used? Would you provide some example? 
At the moment it's used only in `NearestCities`

Reply:
And here it will be used in favourote cities. It is the same component but with different button only